test/runtests.jl

```diff
@@ -25,8 +25,8 @@ for test in [
         failed_tests += 1
         println(output)
         println(outbuf)
-        println(UTF8String(output))
-        println(UTF8String(outbuf))
+        println(String(output))
+        println(String(outbuf))
```

Comment:
```
julia> using Compat

julia> import Compat.String

julia> io = IOBuffer()
IOBuffer(data=UInt8[...], readable=true, writable=true, seekable=true, append=false, size=0, maxsize=Inf, ptr=1, mark=-1)

julia> print(io, "adcdsafea")

julia> outbuf = takebuf_array(io)
9-element Array{UInt8,1}:
 0x61
 0x64
 0x63
 0x64
 0x73
 0x61
 0x66
 0x65
 0x61

julia> String
Union{ASCIIString,UTF8String}

julia> String(outbuf)
ERROR: MethodError: `convert` has no method matching convert(::Type{ByteString}, ::Array{UInt8,1})
This may have arisen from a call to the constructor Union{ASCIIString,UTF8String}(...),
since type constructors fall back to convert methods.
Closest candidates are:
  call{T}(::Type{T}, ::Any)
  convert(::Type{ByteString}, ::AbstractString)
  convert{T<:Union{ASCIIString,UTF8String},S<:Union{Char,Int32,UInt32}}(::Type{T<:Union{ASCIIString,UTF8String}}, ::AbstractArray{S<:Union{Char,Int32,UInt32},1})
  ...
 in call at ./essentials.jl:56
```